.github/workflows/macos.yaml

```diff
@@ -3,36 +3,42 @@ name: macos
 on: [push, pull_request]
 
 jobs:
-
   macos-build:
-    runs-on: ${{ matrix.macos-version }}
+    runs-on: ${{ matrix.os.os }}
     strategy:
       fail-fast: false
       matrix:
-        macos-version:
-          - 'macos-latest'
+        os:
+          - {arch: clang_64, os: macos-13}
+          - {arch: arm64, os: macos-latest}
 
     steps:
       - name: Sync repository
         uses: actions/checkout@v2
 
-      - name: Cache Qt
-        id: cache-qt
-        uses: actions/cache@v1  # not v2!
-        with:
-          path: '${{ github.workspace }}/qt_installation/'
-          key: ${{ runner.os }}-QtCache
+      - name: Clean Homebrew locks and cache
+        run: |
+          rm -rf /usr/local/var/homebrew/locks
+          brew cleanup -s
 
-      - name: Install Qt
-        uses: jurplel/install-qt-action@v2.13.0
-        with:
-          version: '5.15.2'
-          host: 'mac'
-          dir: '${{ github.workspace }}/qt_installation/'
-          cached: ${{ steps.cache-qt.outputs.cache-hit }}
-
-      - name: Build Plotjuggler
-        shell: pwsh
-        run: >
-          cmake -B build -DCMAKE_INSTALL_PREFIX=install PlotJuggler;
+      - name: Install dependencies
+        run: |
+          brew update
+          brew install mosquitto zeromq qt@5
```

Comment:
Any reason to not use the same instructions in COMPILE.md to install and link qt5?